app/views/facility_facility_accounts/index.html.haml

```diff
@@ -9,18 +9,26 @@
 %p= text(".main")
 
 %ul.inline
-  %li= link_to t("admin.shared.add", model: FacilityAccount.model_name.human), new_facility_facility_account_path, :class => "btn-add"
+  %li
+    = link_to text("admin.shared.add", model: FacilityAccount.model_name.human),
+      new_facility_facility_account_path, class: "btn-add"
 
 - if @accounts.empty?
-  %p.notice= t(".notice")
+  %p.notice= text(".notice")
```

Comment:
Since this is now `text`, do we need the leading period? Same note for below in `.inactive`.